resources/com/mesosphere/global/terraform-file-dcos-terraform-test-examples/aws-0.2.x/Terraformfile

```diff
@@ -0,0 +1,7 @@
+{
+
+  "dcos-terraform/windows-instance/aws": {
+    "source": "git::https://github.com/dcos-terraform/terraform-aws-windows-instance.git?ref=support/0.2.x"
```

Comment:
instead of adding this file we should do an initial release of windows instance. Any objections @sebbrandt87 ?

Reply:
I would say we could release it with a 0.0.x version, and then we could get rid of this entry here.

Reply:
agreed

Reply:
Hi, @sebbrandt87 do you have any due date? At the current stage it's our roadblock.

Reply:
we can just do it within seconds ;)

Reply:
fixed at 190bc784c1ed8f11a9b48975c603d05a598485df